src/components/footer/Footer.tsx

```diff
@@ -1,2 +1,30 @@
+import facebookIcon from '../../assets/facebook.png';
+import instagmramIcon from '../../assets/instagram.png';
+import linkedinIcon from '../../assets/link.png';
+import FooterIcon from '../FooterIcon/FooterIcon.tsx';
+import Link from '../Link/Link';
+
 import styles from './footer.module.css';
-export const Footer = () => <footer className={styles.footer}>Footer</footer>;
+export const Footer = () => (
+    <footer className={styles.footer}>
+        <div className={styles.footerIcons}>
+            <FooterIcon icon={facebookIcon} url="https://www.facebook.com/" className={styles.fbIcon} />
```

Comment:
It is good that u create a component for icons
but if u will save your icon in `svg` format from figma instead of `png`
with help of SVGR library u can use svg icons as a ReactComponent
ex:
/assets
 --myIcon.svg
 
 ```tsx
 import MyIcon from "./assets/myIcon.svg"
 
 const Component = () => {
   return (
     // ...a lot of jsx
      <MyIcon className="I can accept any props as svg html elem" />
   )
 }
 ```

Reply:
I have errors when creating such a component, because I cannot attach a link to it
I understood that it is necessary to make an svg component here, but this component does not accept parameters, or I really understood what exactly needs to be done here

Reply:
of-course, i clone your project and try to do it myself
you was right according to the [official](https://www.npmjs.com/package/vite-svg-loader) guide of the plugin it is not clear how to make it with help of Vite builder
but i found an answer here
https://stackoverflow.com/questions/70309561/unable-to-import-svg-with-vite-as-reactcomponent

and this is screen what i did:
![Screenshot 2024-04-27 at 09 59 33](https://github.com/Allocene/fe-react-2024/assets/55930010/42dad787-998c-4005-b330-7526c50e85d8)

Please give me know , is it work correctly on your side 😉

Reply:
Hi, yes, this finally worked

Reply:
But as I understand it, I need to create a separate footericon component that will accept the svg component and then just import it into the footer?

Reply:
I think I understood what was wanted from me)